### ISSUE TITLE
Release 1.0.3 missing agent.example.yaml

### ISSUE THREAD

**Issue by s0meguy1** [evilsocket/shieldwall#1]
It didn't prevent me from setting up in the end, but using the lazy "I'll just copy the code from the quick start guide to get it working" failed on the install.sh step because it couldn't find agent.example.yaml (I did change the version from 1.0.0 in the quickstart to 1.0.3).

Just figured I'd let you know so you don't get more complaints.

Been looking for something like this for a while, love it!

**Comment by s0meguy1** [comment]
Oh I see, after you create an agent on shieldwall.me, you end up blowing out the example completely with a new one. Sorry I was writing this as I was going (probably should have waited. In any case it may be something to mention for others in the wiki just so you don't get dummies like me posting issues 😂😂

**Comment by evilsocket** [comment]
i'm not sure i understand ... on shieldwall.me, when you create an agent you're provided with the configuration to copy/paste, what do you need the example yaml for?

**Comment by s0meguy1** [comment]
Sorry if that was confusing. I downloaded the 1.0.3 release, unzipped, ran install.sh and got errors. I realized it was because the install.sh looks for config.yaml or agent.example.yaml, which weren't included in the 1.0.3 release zip:
![image](https://user-images.githubusercontent.com/25542926/110469054-21ce6400-80a7-11eb-92b3-88c7a02a504e.png)

I resolved it with no issues, I just wanted to let you know. Thanks!
